src/db/user.rs

```diff
@@ -1,22 +1,26 @@
 use std::collections::btree_map::Entry;
 use std::collections::BTreeMap;
 
+use gluer::metadata;
 use serde::{Deserialize, Serialize};
 
 use super::Books;
 use crate::error::{Error, Result};
 use crate::mail::account_is_valid;
 
 /// Data object for a user.
+#[metadata]
 #[derive(Debug, Clone, Deserialize, Serialize)]
 #[cfg_attr(test, derive(PartialEq))]
 #[serde(default)]
 pub struct User {
     pub account: String,
     pub forename: String,
     pub surname: String,
+    #[meta(into = Option<String>)]
     #[serde(skip_serializing_if = "String::is_empty")]
     pub role: String,
+    #[meta(into = Option<bool>)]
```

Comment:
Same here, `#[meta(default = true)]` is just easier than checking `user.may_borrow === true || user.may_borrow == null` everywhere in the ts code.